integration-tests/tx-rollup-transfer-ticket.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { OperationContentsAndResultTransferTicket, RpcClient } from "@taquito/rpc";
+import { Protocols } from "@taquito/taquito";
+import { TransferTicketParams } from "taquito/src/operations/types";
+import { CONFIGS } from "./config";
+import { InMemorySigner } from "@taquito/signer"
+import { ticketsWithdrawal } from "./data/tickets_withdrawal_contract";
+import { ticketsDeposit } from "./data/tickets_deposit_contract";
+import { HttpBackend } from "@taquito/http-utils";
+
+
+CONFIGS().forEach(({ lib, setup, protocol, txRollupDepositContract, txRollupWithdrawContract }) => {
+  const Tezos = lib;
+  const mondaynet = protocol === Protocols.ProtoALpha ? test: test.skip;
+
+  describe(`mondaynet test TransferTicket operation`, () => {
+    beforeEach(async (done) => {
+      await setup();
+      done();
+    })
+    mondaynet("transfer tickets L2 to L1 final step in toru node rollup back to L1", async (done) => {
+      try {
+        const backend = new HttpBackend
+        const checkFinalized = async () => {
+          // check L2 finalization
+          const req = await backend.createRequest<{metadata: {finalized: boolean}}>({ url: 'http://mondaynet.ecadinfra.com:9999/block/head', method: 'GET'})
+          expect(typeof req.metadata.finalized).toEqual('boolean')
+
+          if (req.metadata.finalized) {
+            const signer = await InMemorySigner.fromSecretKey(process.env['TX_ROLLUP_TICKETS_OWNER_SECRET'] || '')
+            await Tezos.setProvider({signer})
+            const estimateParams: TransferTicketParams = {
+              ticketContents: { "string": "foobar" },
+              ticketTy: { "prim": "string" },
+              ticketTicketer: txRollupDepositContract,
+              ticketAmount: 1,
+              destination: txRollupWithdrawContract,
+              entrypoint: "default"
+            };
+            const estimate = await Tezos.estimate.transferTicket(estimateParams);
+
+            const params: TransferTicketParams = {
+              ...estimateParams,
+              fee: estimate.burnFeeMutez,
+              gasLimit: estimate.gasLimit,
+              storageLimit: estimate.storageLimit
+            };
+
+            const res = await Tezos.contract.transferTicket(params);
+            const results = res.results
```

Comment:
we should wait for the operation to be included in a block with `await res.confirmation()`
Then adding an expect statement that the status is `applied`